src/components/HelperErrorText/HelperErrorText.stories.mdx

```diff
@@ -87,10 +88,27 @@ import DSProvider from "../../theme/provider";
 
 ## Accessibility
 
-In the "invalid" state when `isInvalid` is set to true, the text passed to the
-`invalidText` prop will be presented to screen readers be default. If you need
-to turn this off, set the `ariaLive` prop to `"off"`. This is used in DS "form"
-components for invalid form submissions or incorrect values entered.
+In the Reservoir Design System (DS), the `HelperErrorText` component is always
+associated with a related element by using the `aria-describedby` attribute in
+the associated element. In this way, the content of the `HelperErrorText` is
+made available to screenreaders.
+
```

Comment:
Maybe write more to explain the example and how the input field is associated with the helper text "This is error text :(".